pkgs/development/tools/rust/rust-analyzer/default.nix

```diff
@@ -11,14 +11,14 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "rust-analyzer-unwrapped";
-  version = "2022-04-11";
-  cargoSha256 = "sha256-pc8S+5g67vN9170ABiHGe1t4Go0F63UiOeLd71USYCI=";
+  version = "2022-05-02";
+  cargoSha256 = "sha256-uZCUruIBTNTKYWYb8xQgJ6FsKlRi+Sh5n7m7aVk+hHQ=";
 
   src = fetchFromGitHub {
     owner = "rust-analyzer";
```

Comment:
Should this also be updated?

Reply:
Yes, though Github will redirect you - the redirect was breaking `update.sh` though -- I figure that when someone fixes up the `update.sh` for the vscode extension they should probably hit this bit too if they can.  I tried that last night but after 2h of waiting for node2nix I cancelled the attempt.